app/controllers/items_controller.rb

```diff
@@ -184,6 +223,9 @@ def release_hold
   end
 
   def embargo_update
+    # TODO: this could be a more granular permission request, i.e.
+    #apo = @object.admin_policy_object
+    #@object.can_manage_embargo?(current_user.roles(apo.pid))
```

Comment:
can we lump all the TODOs of this sort into a github issue with references to the code?  these all seem like suggestions for functional changes.  they might be minor changes that would be easily agreed upon, but they seem to warrant discussion (both for whether they're desirable in principle, and for whether we want to bother implementing them even if they are).